### DISCUSSION TITLE
Berry Print Statement

### DISCUSSION THREAD

**Discussion by trlafleur** [arendst/Tasmota#19555] [Q&A] [answered]
I noticed during debugging of my berry code, that the print statement stopped displaying after a period of time. 

Its seems to be an intermittent issue...

13.1.0.3

**Comment by sfromis** [comment]
Assuming that you are talking specifically about what you see in the Berry REPL console....

As the web frame refresh is managed by JavaScript code running in the browser, things may not get updated when you are not scrolled to the end, or (depending on browser) maybe even if you are not viewing the tab.

Depending on what you actually want to see, and by what tools, other methods of logging may be preferable.

**Reply by trlafleur** [reply, marked as answer]
Moving to Chrome solved this issue...
